### ISSUE TITLE
clarify whether `label` is a required field for entities

### ISSUE THREAD

**Issue by stefanprobst** [acdh-oeaw/InTaVia-Backend#10]
currently, the `label` field on entity search results is optional - the only "required" field is `id` - is `label.default` (and `kind`) guaranteed to be populated?

**Comment by stefanprobst** [comment]
similar question for `EntityEventRelation.entity` and `EntityEventRelation.role`

**Comment by sennierer** [comment]
yes, left that optional for the testing phase (to prevent validation errors), but I guess labels and ids should be required in general.

**Comment by sennierer** [comment]
leaving that for the time being open. in case that is a problem for the interfaces create from the API specs, please comment on the issue
